### PR TITLE
Handle direct Ether transfers in `Autoclaim`

### PR DESCRIPTION
**Pull request by sporejack**
# Issue
#260 

# Changes
See [comment](https://github.com/tracer-protocol/perpetual-pools-contracts/issues/260#issuecomment-985136707) on the associated issue.

### CODE REVIEW

File: contracts/implementation/AutoClaim.sol [hunk 1]
```diff
@@ -162,6 +162,14 @@ contract AutoClaim is IAutoClaim, Initializable {
         return request.updateIntervalId > 0 && request.updateIntervalId < currentUpdateIntervalId;
     }
 
+    receive() external payable {
```
Comment (Paul4912) [inline contracts/implementation/AutoClaim.sol:165]: why did removing function fix the lint error? Does it not need the function keyword here?

Comment (sporejack) [reply contracts/implementation/AutoClaim.sol:165]: I guess not -- I can track down some documentation if you'd like?

Comment (Paul4912) [reply contracts/implementation/AutoClaim.sol:165]: https://blog.soliditylang.org/2020/03/26/fallback-receive-split/

example here didn't have function keyword either

**Review by Paul4912 — APPROVED** [review]

File: test/AutoClaim/transfer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        shortToken = result.shortToken
+        longToken = result.longToken
+
+        await token.approve(pool.address, amountMinted)
+    })
+
+    context("When Ether is transferred without calldata", async () => {
+        it("Reverts", async () => {
+            const transferTo: any =
+                autoClaim.address /* TODO: should be `Address`, not `any` */
```
Comment (mynameuhh) [inline test/AutoClaim/transfer.spec.ts:87]: Should this be fixed in this PR, or a separate one?

File: test/AutoClaim/transfer.spec.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            }
+
+            await expect(someUser.sendTransaction(transferTx)).to.be.reverted
+        })
+    })
+
+    context("When Ether is transferred with calldata", async () => {
+        it("Reverts", async () => {
+            const transferTo: any =
+                autoClaim.address /* TODO: should be `Address`, not `any` */
```
Comment (mynameuhh) [inline test/AutoClaim/transfer.spec.ts:103]: Same as above

File: test/AutoClaim/transfer.spec.ts (hunk 2 above)
Comment (sporejack) [reply test/AutoClaim/transfer.spec.ts:87]: These are Typescript-level issues: Ethers should expose the `Address` type in the root of the module but it doesn't. I can remove the comments if you'd prefer?

Comment (mynameuhh) [reply test/AutoClaim/transfer.spec.ts:87]: Nah all good.

**Review by mynameuhh — APPROVED** [review]

**Pull request merged**